backend/census_historical_migration/workbooklib/excel_creation_utils.py

```diff
@@ -62,7 +57,7 @@ def set_uei(Gen, wb, dbkey):
     if g.uei:
         set_range(wb, "auditee_uei", [g.uei])
     else:
-        g.uei = "BADBADBADBAD"
+        g.uei = "BADBADBADBAD"  # FIXME: Shouldn't this raise an exception here?
```

Comment:
Now it should, yes. Although, it will be 100% true for all audits before 2022. So, we have to have a strategy.

We can discuss. It should not be `BADBADBADBAD`, as you suspect.